testdata/case-2-error-microerror-Error-depth-1-Maskf.golden

```diff
@@ -5,7 +5,7 @@
 	"annotation": "test annotation",
 	"stack": [
 		{
-			"file": "--REPLACED--/github.com/giantswarm/microerror/json_test.go",
+			"file": "--REPLACED--/json_test.go",
```

Comment:
It is `/root/project/json_test.go` in the orb container. See https://circleci.com/gh/giantswarm/microerror/118?utm_campaign=vcs-integration-link&utm_medium=referral&utm_source=github-build-link.